### ISSUE TITLE
Magento 2

### ISSUE THREAD

**Issue by dominik04u** [blopa/Magento-Chatbot#120]
I run your plugin for Magento 2 but there is a problem:
```
 Werules\Chatbot\Block\Webhook\Messenger
                Missed required argument request in parent::__construct call. File: /var/www/html/application/app/code/Werules/Chatbot/Block/Webhook/Messenger.php
Total Errors Count: 1


  [Magento\Framework\Validator\Exception]
  Error during compilation
```

**Comment by blopa** [comment]
Hello @dominik04u. Thanks for reporting it. I still have to handle some exceptions. 

Did you configured the Api Token in the Magento backend?

**Comment by blopa** [comment]
Fixed on version 1.0.1 https://github.com/blopa/Magento-Chatbot/releases/tag/m2_v1.0.1
